csharp/roslyn-sdk/Tutorials/MakeConst/MakeConst/MakeConstAnalyzer.cs

```diff
@@ -28,10 +28,11 @@ public class MakeConstAnalyzer : DiagnosticAnalyzer
 
         public override void Initialize(AnalysisContext context)
         {
+            context.ConfigureGeneratedCodeAnalysis(GeneratedCodeAnalysisFlags.Analyze | GeneratedCodeAnalysisFlags.None);
+            context.EnableConcurrentExecution();
             // See https://github.com/dotnet/roslyn/blob/master/docs/analyzers/Analyzer%20Actions%20Semantics.md for more information
             // <SnippetRegisterNodeAction>
             context.RegisterSyntaxNodeAction(AnalyzeNode, SyntaxKind.LocalDeclarationStatement);
-            // </SnippetRegisterNodeAction>
```

Comment:
I think removing this line produces a bug.
This code snippet is referenced [here](https://github.com/dotnet/docs/blob/master/docs/csharp/roslyn-sdk/tutorials/how-to-write-csharp-analyzer-code-fix.md)

If you take a look at the raw of the previous mentioned file you will find:
![image](https://user-images.githubusercontent.com/31348972/64065543-29d65980-cc0f-11e9-9e0a-599950fa0822.png)
The highlighted part after the `#` states that only part of the file will be referenced, and that part is that line between:

```csharp
// <SnippetRegisterNodeAction>
```

and

```csharp
// </SnippetRegisterNodeAction>
```

Reply:
This was change by VS with the new analyzer codefix to add, I will fix here and file bug against Roslyn.
```
            context.ConfigureGeneratedCodeAnalysis(GeneratedCodeAnalysisFlags.Analyze | GeneratedCodeAnalysisFlags.None);
            context.EnableConcurrentExecution();
```